include/sound/sof/ext_manifest.h

```diff
@@ -61,6 +61,18 @@ enum sof_ext_man_elem_type {
 	SOF_EXT_MAN_ELEM_WINDOW			= SOF_IPC_EXT_WINDOW,
 	SOF_EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	SOF_EXT_MAN_ELEM_DBG_ABI		= SOF_IPC_EXT_USER_ABI_INFO,
```

Comment:
a typo in the commit message? Shouldn't it be "CAVS_LPRO" on both occasions? And if yes, maybe would be enough to write "Currently CAVS_PRO is only supported, it has default clock in CAVS platform." Although I don't quite understand what "default clock in CAVS platform" actually means

Reply:
Thank you for checking typo! I updated commit text as well